src/components/FeatureFlagModal/FeatureFlagModal.tsx

```diff
@@ -209,6 +210,14 @@ export default function FeatureFlagModal() {
           label="Enables JSON-RPC tracing"
         />
       </FeatureFlagGroup>
+      <FeatureFlagGroup name="Tokens">
```

Comment:
Should this be in a separate PR?

Reply:
i can do in sep pr yes, but i did it so we had no more unused things

Reply:
https://github.com/Uniswap/interface/pull/5511